Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md

```diff
@@ -142,6 +142,39 @@ Follow these steps to schedule an instance.
 
   Besides the user interface, you can automatically schedule instances using the instances API with a [POST or PUT](https://www.ctl.io/api-docs/cam/#application-lifecycle-management-instances-api) request.
 
+### Protecting Instance Shutdown/Termination
+
+Avoid accidental operations on particular instances. *Instance Protection Flag* allows to protect an instance for manual **shutdown** or **termination**. Instance still being affected by *scheduled termination or shutdown*. 
+
+*Instance Protection Flag* forbid operators to run shutdown/terminate operations on API/UI level, also affecting at **provider level** if implements this feature. 
+
+*Instance Protection Flag* can be setted on **Deployment Policy Level** and on **Instance Deployment Level**. 
+
+Follow these steps to enable instance protection on **deployment policy**:
+
+1. From the boxes page, click **New**
+2. Select **Deployment Policy**
+3. In the New Deployment Policy Box dialog, enable the **Manual Shutdown Protection** or **Manual Terminate Protection** toggle under **expiration** dropdown.
+4. When done, click **Save**
+
+**Note:** Instances affected by created policy box will propagate the protection configuration flags, preventing operators to disable them.
+
+Follow these steps to protect an instance on **deployment**:
```

Comment:
```suggestion
Follow these steps to protect an instance at **deployment** time:
```
You should mention that for a deployed instance you will be able to change these toggles by editing its details